doc/themes/the-things-stack/assets/css/theme.scss

```diff
@@ -1,13 +1,15 @@
 @charset "utf-8";
 
 @import url('https://fonts.googleapis.com/css?family=Lato:400,700&display=swap');
+@import url('https://rsms.me/inter/inter.css');
 
 @font-face {
```

Comment:
You're adding font files, but not defining a `font-face` for Inter.

Reply:
This imports a whole CSS, which includes the `font-face`.